### PR TITLE
Kata 

### PR DESCRIPTION
**Pull request by Elijahthe3rd**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseURL =  "/tech-department/"
+baseURL =  "/"
```
Comment (sheenarbw) [inline config.toml]: No.

File: content/katas/_index.md [hunk 2]
```diff
@@ -98,10 +98,148 @@ rotate("hello",-2) => lohel
...
+
+**7.**
+Write a function that takes in number as argument and draws a Hollow pyramid triangle above using the " * " characters . If the input is 4, then the output is a traingle shaped as pyramid with height 4
+```
+// JavaScript
+let triangle(5) => 
+expected output: HOLLOW RIGHT ANGLE TRIANGLE AND FOR ITS MIRROR.      
+    
+    *
+   * *
```
Comment (sheenarbw) [inline content/katas/_index.md]: Ok, we need to do another git feature branching workshop. You are mixing your stuff up.

I need you to know how to use git like a professional before I can consider you competent. I need you to demonstrate your competence

File: content/topics/intro-to-relational-databases/_index.md [hunk 3]
```diff
@@ -1,15 +1,99 @@
...
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
+
```
Comment (sheenarbw) [inline content/topics/intro-to-relational-databases/_index.md]: Again. Git. Learn it

File: content/topics/intro-to-relational-databases/_index.md [hunk 4]
```diff
@@ -1,15 +1,99 @@
...
+                ![alt text](https://miro.medium.com/max/1512/1*LEksJP5OtS8GEBdd2Jy4WQ.png)
+
+**Best practices**  
+```
+*(N.B)* 
+If your using VIM or VI as your code Editor...
 
-- [Intro to SQL for data science](https://www.datacamp.com/courses/intro-to-sql-for-data-science)
-- [Intermediate SQL](https://www.datacamp.com/courses/intermediate-sql)
+Always do not forget to include your .gitignore file
```
Comment (sheenarbw) [inline content/topics/intro-to-relational-databases/_index.md]: I already gave you feedback on this

File: content/topics/intro-to-relational-databases/_index.md [hunk 5]
```diff
@@ -1,15 +1,99 @@
...
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
```
Comment (Wonder13oy) [inline content/topics/intro-to-relational-databases/_index.md]: I do not understand why this topic is in this branch. Doesn't this need it's own branch.

File: content/katas/_index.md [hunk 6]
```diff
@@ -98,10 +98,117 @@ rotate("hello",-2) => lohel
...
+*
+*
+*
+*****
+``` 
+## Ceaser cipher
 
-## fibonacci
+             
+## fibonacci sequence
```
Comment (Wonder13oy) [inline content/katas/_index.md]: Is it possible to have each kata in its own folder. It would make it easier for browsing and finding what you need.

**Pull request closed without merging**